### PR TITLE
docs: add an exception for accepting contributions

### PR DESCRIPTION
**Pull request by mgechev**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -119,6 +119,11 @@ Before you submit your Pull Request (PR) consider the following guidelines:
 
 11. In GitHub, send a pull request to `angular:master`.
 
+### Reviewing a Pull Request
+
+The Angular team reserves the right not to accept pull requests from community members who haven't been good citizens of the community. Such behavior includes not following the [Angular code of conduct](https://github.com/angular/code-of-conduct) and applies within or outside of Angular managed channels.
+
+If the team is not familiar with any violations of the code of conduct, they will proceed with a review.
```
Comment (IgorMinar) [inline CONTRIBUTING.md:126]:
```suggestion
If the team is not familiar with any violations of the code of conduct, we will proceed with a review.
```

**Review by IgorMinar — APPROVED** [review]
lgtm - one small nit.

File: CONTRIBUTING.md (hunk 1 above)
Comment (mgechev) [reply CONTRIBUTING.md]: I removed this sentence based on feedback from @jelbourn and @aikidave.

**Review by josephperrott — APPROVED** [review]
LGTM

**Review by jelbourn — APPROVED** [review]
LGTM

**Comment by AndrewKushnir** [comment]
Hey @mgechev, it looks like this PR was approved. If it's still relevant, please let us know, so that we can add it to the merge queue (or you can add it yourself by adding the "target: patch" and "action: merge" labels). Thank you.

**Pull request closed without merging**